packages/trader/src/AppV2/Components/ForwardStartingBanner/__tests__/forward-starting-banner.spec.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { toMoment } from '@deriv/shared';
+import ForwardStartingBanner from '../forward-starting-banner';
+
+const mocked_now = Math.floor(Date.now() / 1000);
```

Comment:
can we do a fixed date instead of date.now

Reply:
Let me check, but in that case I need to mock the returned value of hasForwardContractStarted contract as it's comparing start date with Date now
![Uploading Screenshot 2024-09-25 at 10.34.49 AM.png…]()

Reply:
<img width="636" alt="Screenshot 2024-09-25 at 10 34 49 AM" src="https://github.com/user-attachments/assets/03b5e687-27b0-48cb-8bbe-f0c2c64d89b7">

Reply:
you should mock Date.now

Reply:
Applied